runsc/container/container.go

```diff
@@ -311,6 +311,13 @@ func New(conf *boot.Config, args Args) (*Container, error) {
 	if isRoot(args.Spec) {
 		log.Debugf("Creating new sandbox for container %q", args.ID)
 
+		if args.Spec.Linux == nil {
+			args.Spec.Linux = &specs.Linux{}
+		}
+		if args.Spec.Linux.CgroupsPath == "" {
```

Comment:
For tests it will be easier to just skip this part. There are many different errors that the tests can return and they already run inside bazel sandbox. So this becomes:

		// Don't force the use of cgroups in tests because they lack permission to
		// do so.
		if args.Spec.Linux.CgroupsPath == "" && !conf.TestOnlyAllowRunAsCurrentUserWithoutChroot {
			args.Spec.Linux.CgroupsPath = "/" + args.ID
		}

And you can remove one of the `case` below.